samples/word/45-shapes/manage-geometric-shapes.yaml

```diff
@@ -0,0 +1,148 @@
+order: 2
+id: word-shapes-manage-geometric-shapes
+name: Manage geometric shapes
+description: Shows how to work with geometric shapes.
+host: WORD
+api_set:
+    WordApiDesktop: '1.2'
+script:
+    content: |
+        document.getElementById("insert-geometric-shape").addEventListener("click", () => tryCatch(insertGeometricShape));
+        document.getElementById("get-geometric-shapes").addEventListener("click", () => tryCatch(getGeometricShapes));
+        document.getElementById("get-heptagons").addEventListener("click", () => tryCatch(getSpecificGeometricShapes));
+        document.getElementById("get-first-geometric-shape").addEventListener("click", () => tryCatch(getFirstGeometricShape));
+        document.getElementById("get-first-heptagon").addEventListener("click", () => tryCatch(getFirstHeptagon));
+
+        async function insertGeometricShape() {
+          await Word.run(async (context) => {
+            // Inserts a geometric shape at the beginning of the selection.
+            const selection: Word.Range = context.document.getSelection();
+            const shapeOptions: Word.InsertShapeOptions = {
+              height: 120,
+              width: 120
+            };
+            selection.insertGeometricShape(Word.GeometricShapeType.heptagon, shapeOptions);
+            await context.sync();
+
+            console.log("Inserted a heptagon.");
+          });
+        }
+
+        async function getGeometricShapes() {
+          await Word.run(async (context) => {
+            // Gets the geometric shapes from the document body.
+            const geometricShapes: Word.ShapeCollection = context.document.body.shapes.getByTypes([
+              Word.ShapeType.geometricShape
+            ]);
+            geometricShapes.load();
+            await context.sync();
+
+            console.log("Geometric shapes found in the document body:", geometricShapes);
+          });
+        }
+
+        async function getSpecificGeometricShapes() {
```

Comment:
Consider having two `addShape` methods so the `getFirst` and `getSpecific` methods return different things.